examples/sol-anchor-contract/Anchor.toml

```diff
@@ -0,0 +1,10 @@
+[programs.devnet]
+example_sol_anchor_contract = "4ACQwuQt2ChnPRUV8KwuhVqJvEdusWd3i8JAngHDkp91"
+
+[provider]
+cluster = "devnet"
+wallet = "/home/yunhao/pyth/wallet/pair.json"
+
+[scripts]
+install = "npm i chai mocha ts-mocha @types/bn.js @types/chai @types/mocha typescript prettier @project-serum/anchor"
```

Comment:
Typically to install you want to do `npm install`. This will go to your `package.json` and get the right packages so you don't have to specify all of them here.